lib/screens/history/widgets/history_file_item.dart

```diff
@@ -641,15 +641,12 @@ class _HistoryFileItemState extends State<HistoryFileItem> {
   }) {
     return InkWell(
       onTap: onTap,
-      child: CommonUtilityFunctions()
-              .isFileDownloadAvailable(widget.fileTransfer!.date!)
```

Comment:
Why are we removing this check? The download icon should not be available if the download is expired

Reply:
@Sardar1208 I moved the handle-checking method to the above widget

Reply:
Okay